src/banking/sheba.rs

```diff
@@ -0,0 +1,356 @@
+use crate::impl_trait_for_string_types;
+use lazy_static::lazy_static;
+use regex::Regex;
+
+lazy_static! {
+    static ref SHEBA_CODE_PATTERN: Regex = Regex::new(r#"^IR[0-9]{24}$"#).unwrap();
+}
+
+#[derive(Debug)]
+pub struct ShebaInfo {
+    nickname: &'static str,
```

Comment:
I would like to open a discussion here, I don't know the answer myself!

For such a thing, which has a dynamic concept, does it make sense to have `&'static str` or `String`? and Why?
By "dynamic concept" I mean something which can be filled dynamically, from database entries, or user inputs.

@kianenigma @robatipoor

Reply:
I think there is no need now if we need it in the future we will make the necessary changes

Reply:
No, I didn't mean this code is wrong or correct.
I asked generally. Which one is better and why?

Reply:
I think neither is better than the other, each has its own uses